### DISCUSSION TITLE
Welcome to discord-js-community-bot Discussions!

### DISCUSSION THREAD

**Discussion by SneakyHub** [SneakyHub/discord-js-community-bot#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Microwave6443** [comment]
gonna start working on this again soon

**Comment by LukeBruh** [comment]
Do you have a server? Could you send the link?

**Reply by SneakyHub** [reply]
You can join our discord.. https://sneakyhub.com/discord

Then you can create a server from there.
